src/migrations/20180516164000-endpoint-specific-apps.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const configJsonPath = process.env.CONFIG_MOUNT_POINT;
+
+exports.up = function (knex, Promise) {
+	return new Promise((resolve, reject) => {
+		if (!configJsonPath) {
+			resolve({});
+		}
+		fs.readFile(configJsonPath, (err, data) => {
+			try {
+				const parsed = JSON.parse(data.toString());
+				resolve(parsed);
+			} catch(e) {
+				resolve({});
```

Comment:
I suggest at least adding a console.log warning here, as it would be quite rare not to have a config.json at all (and currently other things would break in that case)

Reply:
Sounds good!